portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,39 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  background-color: white;
+  color:white;
+  font-family: 'Staatliches', cursive;
+  letter-spacing: 3px;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden; 
+}
+h1 {
+  padding-top: 10px;
+  font-size: 10.0em;
+}
+h2 {
+  font-size: 3.0em;
+  color: white;
+}
+.navbar-inverse .navbar-nav>.active>a, .navbar-inverse .navbar-nav>.active>a:focus, .navbar-inverse .navbar-nav>.active>a:hover {
+  background-color: #ed5565;
+}
+#overlay {
+  background: rgba(0, 0, 0, 0.76);
+  height: 1000px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.title {
+  padding-bottom: 10px;
+  width: 1100px;
+  margin-right: auto;
+  margin-left: auto;
+  position: relative;
+  top: 250px;
 }
+
+#qualif {
+  font-size: 3em;
+  color:#ff2a4a;
+}
```

Comment:
we should add new line at the end of every file.

Reply:
Sure, will do it from next time, thanks.